CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # activerecord-forbid_implicit_checkout
 
+## v2.0.0
+- Add rails 6.1 support
+- Drop rails < 6 support
```

Comment:
This isn't accurate anymore

Reply:
Changed the version to 1.1 since there is nothing breaking now